src/main/java/com/my/relink/domain/item/exchange/ExchangeItem.java

```diff
@@ -103,4 +103,7 @@ public void update(String name, String description, Category category,
         this.desiredItem = desiredItem;
         this.deposit = deposit;
     }
+    public void delete(Boolean isDeleted) {
+        this.isDeleted = isDeleted;
+    }
```

Comment:
```suggestion
    public void delete() {
        this.isDeleted = true;
    }
```
이 메서드 호출하면 무조건 true 값 만 넘겨주니 이런식으로 쓸수 있을거 같애요 ~!!

Reply:
아 그렇게 쓰면 훨씬 편하겠네요 감사합니다.